crawler/options.go

```diff
@@ -20,7 +21,7 @@ type options struct {
 // defaults are the default crawler options. This option will be automatically
 // prepended to any options you pass to the crawler constructor.
 var defaults = func(o *options) error {
-	o.protocols = []protocol.ID{"/ipfs/kad/1.0.0"}
+	o.protocols = amino.Protocols
 	o.parallelism = 1000
 	o.connectTimeout = time.Second * 5
 	o.perMsgTimeout = time.Second * 5
```

Comment:
Should these be defined in `defaults.go` as well, even if not exported?

Reply:
These seem to be specific  to crawler and really part of "Amino DHT contract", so could 
I'm going to merge thus PR as-is, we can figure out what to do remaining magic numbers in follow-up PR.